infrastructure/prod.tfvars

```diff
@@ -1,4 +1,4 @@
-idam_api_url = "https://prod-idamapi.reform.hmcts.net:3511"
+idam_api_url = "https://prod-idamapi.reform.hmcts.net"
```

Comment:
It would be worth adding in description of PR why port is not required any more

Reply:
I've added more words to the description